src/components/Questionnaire/QuestionTypes/MedicationRequestQuestion.tsx

```diff
@@ -412,12 +412,7 @@ export function MedicationRequestQuestion({
   });
 
   return (
-    <div
-      className={cn(
-        "space-y-4",
-        medications.length > 0 ? "md:max-w-fit" : "max-w-4xl",
-      )}
-    >
+    <div className="space-y-4 w-full">
```

Comment:
This is not responsive in large screens, this starts breaking something which wasn't already breaking in existing version.

<img width="2056" height="317" alt="image" src="https://github.com/user-attachments/assets/c9a3ac27-4ae0-47c1-828c-a6438b29ac7d" />


<img width="1767" height="455" alt="image" src="https://github.com/user-attachments/assets/4c2a95bf-67b5-4f82-8cc7-4b0072ed3808" />